src/vastgoed/meta/model.json

```diff
@@ -0,0 +1,110 @@
+{
+  "vastgoed_id": {
+    "type": "SERIAL",
+    "properties": {
+      "internal": true
+    }
+  },
+  "vhe_id": {
+    "type": "String",
+    "properties": {
+      "primary_key": true,
+      "index": true,
+      "unique": true,
+      "null": false,
+      "length": 16
+    }
+  },
+  "object_id": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 16
+    }
+  },
+  "status": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 256
+    }
+  },
+  "portefeuille": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 256
+    }
+  },
+  "grex": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 256
+    }
+  },
+  "object_type": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 256
+    }
+  },
+  "object_naam": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 512
+    }
+  },
+  "vhe_adres": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 512
+    }
+  },
+  "object_eigendomsstatus": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 128
+    }
+  },
+  "vhe_eigendomsstatus": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 128
+    }
+  },
+  "bag_pand_id": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 32
+    }
+  },
+  "bag_pand_geometrie": {
+    "type": "Geo.Polygon",
+    "properties": {
+      "index": true,
+      "null": true,
+      "geo_find": true
+    }
+  },
+  "bag_verblijfsobject_id": {
+    "type": "String",
+    "properties": {
+      "null": false,
+      "length": 32
+    }
+  },
+  "bag_verblijfsobject_geometrie": {
+    "type": "Geo.Point",
+    "properties": {
+      "index": true,
+      "null": true
+    }
+  }
+}
```

Comment:
Yeah..... Don't agree with writing custom language to build Python models. I get the idea. But creating a specific language is hardly ever a good idea. The problem is that the benefits in configuration are minor compared to the loss in maintainability.

Reply:
Okay. On the other hand it is a direction we are going (see: https://github.com/Amsterdam/schemas and https://github.com/Amsterdam/GOB-Core/blob/master/gobcore/model/gobmodel.json for instance). I think this is not the right place to discuss that, but please join in elsewhere.

Reply:
Okay. This is a direction we are going in (see for instance https://github.com/Amsterdam/schemas/blob/master/bouwdossiers/bouwdossiers.schema.json and https://github.com/Amsterdam/GOB-Core/blob/master/gobcore/model/gobmodel.json) where the model definition (as single source of truth :-) ) is declared, not built in code. Please join in in the discussions.